sources/api/migration/migrations/v1.20.0/update-containerd-config-template-path/src/main.rs

```diff
@@ -0,0 +1,37 @@
+use migration_helpers::common_migrations::{NoOpMigration, ReplaceStringMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+///
+fn run() -> Result<()> {
+    if cfg!(variant_runtime = "ecs") {
```

Comment:
Could you try assigning the migration to a variable, and run that at the end? e.g.

```rust
let migration = if cfg!(variant_runtime = "ecs") {
    ReplaceStringMigration {...}
} else if cfg!(variant_runtime = "k8s") {
     ReplaceStringMigration {...}
} else {
    NoOpMigration
};

migrate(migration)?
```

Reply:
Resolved offline.